src/Deprecated/Engine/Microsoft.Build.Engine.csproj

```diff
@@ -4,7 +4,22 @@
     <NoWarn>$(NoWarn);618;612</NoWarn>
     <GenerateReferenceAssemblySources>true</GenerateReferenceAssemblySources>
     <EnableDefaultItems>false</EnableDefaultItems>
+    <CreateTlb>true</CreateTlb>
```

Comment:
This doesn't actually do anything, does it? Just been carried around from all our old projects?

Reply:
I don't know, I just moved the properties up from the bottom so they were set in one place